go.mod

```diff
@@ -8,11 +8,13 @@ require (
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/cheynewallace/tabby v1.1.1
 	github.com/gizak/termui/v3 v3.1.0
-	github.com/spf13/cobra v1.3.0
+	github.com/spf13/cobra v1.4.0
+	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.10.0
 )
 
 require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.1 // indirect
```

Comment:
we can look workspaces separately so that CI dependencies aren't in the final binary to make it smaller

Reply:
"When compiling packages, build ignores files that end in '_test.go'."

https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies

Reply:
yup, is there any other dependencies could benefit?

Reply:
not sure

Reply:
since we have separate investigation and size of binary is small enough, good to skip for this PR